Lesson_3/math-operations/math-operations.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener('DOMContentLoaded', () => {
+   const firstNumber = Number(prompt('Введіть перше число для розрахунку'));
+   const secondNumber = Number(prompt('Введіть друге число для розрахунку'));
+
+   const resultAddTwoNum = firstNumber + secondNumber;
+   const resultSubTwoNum = firstNumber - secondNumber;
+   const resultMultTwoNum = firstNumber * secondNumber;
+   const resultDivTwoNum = firstNumber / secondNumber;
+
+   if (firstNumber !== firstNumber || secondNumber !== secondNumber) {
```

Comment:
looks Very complicated. You can use isNaN 
```suggestion
   if (isNaN(firstNumber) || isNaN(secondNumber)) {
```